docs/guide/api-environment-classes.md

```diff
@@ -1,4 +1,4 @@
-# Environment API for Frameworks
+# Using `Environment` classes
```

Comment:
The title was same. I put this name for now but probably there's a better name.

Reply:
We were talking about switching from classes to interfaces in the public API. I used JS because that was the name we used in the current API guide. I don't like it either. Maybe Environment instances?